02-IDE_exercises/solutions/06a-project/find_median.py

```diff
@@ -1,19 +1,24 @@
 # Author: Brandon Hoffman
 # Date: 10/25/2020
 # Description: Sorts a list of numbers and returns the median number in the list
+from array import array
 
-def find_median(some_nums):
+
+def find_median(some_nums: list[int]) -> float:
```

Comment:
I believe this is the incorrect syntax. Type hinting a list is done with `from typing import List` and then call it like `List[int]`